docs/concepts/did.md

```diff
@@ -2,3 +2,103 @@
 id: did
 title: DIDs
 ---
+
+import ThemedImage from '@theme/ThemedImage';
+
+A KILT Decentralized Identifier (DID) is a string uniquely identifying each KILT user.
+A DID can be thought of as a container of different keys that are all under the control of the same DID subject.
+For the official W3C DID spec, please visit the [DID Core spec page](https://www.w3.org/TR/did-core/), while for the official KILT DID method specification, please visit the [KILT DID spec page](https://github.com/KILTprotocol/kilt-did-driver/blob/master/docs/did-spec/spec.md).
+
+The simplest type of KILT DID is a **light DID**, called this way because it can be generated and used offline without requiring any Internet connection (hence any connection with the KILT blockchain at all).
```

Comment:
```suggestion
The simplest type of KILT DID is a **light DID**, called this way because it can be generated and used offline without requiring any internet connection (hence any connection with the KILT blockchain at all).
```

Reply:
https://github.com/KILTprotocol/docs/pull/82/commits/27ff8acbe8ca3344d8a85d143427b5423803b27d